offchain/state-server/src/main.rs

```diff
@@ -1,22 +1,18 @@
 // (c) Cartesi and individual authors (see AUTHORS)
 // SPDX-License-Identifier: Apache-2.0 (see LICENSE)
-
-use eth_state_server_lib::config::StateServerConfig;
-use tracing_subscriber::filter::{EnvFilter, LevelFilter};
+mod config;
+use config::Config;
 use types::foldables::authority::rollups::RollupsState;
 
 #[tokio::main]
 async fn main() -> Result<(), Box<dyn std::error::Error>> {
-    let filter = EnvFilter::builder()
-        .with_default_directive(LevelFilter::INFO.into())
-        .from_env_lossy();
-    tracing_subscriber::fmt().with_env_filter(filter).init();
+    let config: Config = Config::initialize_from_args()?;
 
-    let config = StateServerConfig::initialize_from_args()?;
+    logs::configure(&config.logs_config);
 
-    tracing::info!(?config, "starting state server");
+    tracing::info!(?config, "Starting State Server");
```

Comment:
```suggestion
```
This line looks like a duplicate of the following one:

https://github.com/cartesi/rollups-node/blob/e9a9c23299a79a294d2d5ad85099a8bcdae94bfe/offchain/state-server/src/lib.rs#L36

Reply:
@fmoura this one still applies. I'll push a fixup